src/app/purchase/page.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { Controller, useForm } from "react-hook-form";
+import Button from "../_component/common/Button";
+import InputPrice from "../_component/common/InputPrice";
+import ProductCard from "../_component/common/ProductCard";
+import Timer from "../_component/common/Timer";
+import AuctionBanner from "./AuctionBanner";
+import AuctionRanking from "./AuctionRanking";
+import tempImage from "../../public/tempImage.png";
+
+interface Inputs {
+  price: number;
+}
+
+const PurchasePage = () => {
+  const createdAt = new Date("2024-02-16T10:59:59");
+  const deadline = new Date("2024-02-17T16:59:59");
+  const START_PRICE = 10000;
+  const MAX_PRICE = 30000;
+
+  const { control, handleSubmit, setValue, getValues } = useForm<Inputs>();
+
+  const onSubmit = (data: Inputs) => {
+    console.log(data);
+  };
+
+  return (
+    <div
+      style={{
+        margin: "20px auto",
+        display: "flex",
+        flexDirection: "column",
+        gap: "20px"
+      }}>
+      <div style={{ display: "flex", justifyContent: "center" }}>
+        <Timer
+          deadline={deadline}
+          createdAt={createdAt}
+        />
+      </div>
+      <AuctionBanner
+        startPrice={START_PRICE}
+        maxPrice={MAX_PRICE}
+      />
+      <div style={{ display: "flex", justifyContent: "center" }}>
+        <ProductCard
+          titleImage={tempImage}
+          productName="상품명"
+          createDate={new Date("2024-02-26T21:29:00")}
+          price={START_PRICE}
+        />
+      </div>
+      <AuctionRanking maxPrice={MAX_PRICE} />
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Controller
+          control={control}
+          name="price"
+          defaultValue={0}
+          render={({ field }) => (
+            <InputPrice
+              title="입찰 시작가"
+              price={field.value}
+              field={{
+                ...field,
+                onChange: (value: number) => {
+                  setValue("price", value);
+                }
+              }}
+            />
```

Comment:
```suggestion
            <InputPrice<타입, "price">
              title="제안가"
              price={price}
              field={field}
              setValue={() => reset()}
            />
```

reset은 react-hook-form에서 제공하는 옵션이에요! useForm에 추가해서 사용하시면 됩니당
혹시 이렇게 했을 때도 잘 작동되지않는다면 말씀해주세요ㅎㅎ

Reply:
이부분 제가 props명을 setValue로 해서 헷갈리셨던 것 같아요ㅜ reset으로 props명 수정해서 올리도록 하겠습니다!!